app/api/server/v1/groups.js

```diff
@@ -231,6 +232,10 @@ API.v1.addRoute('groups.create', { authRequired: true }, {
 			return API.v1.failure('Body param "extraData" must be an object if provided');
 		}
 
+		if (settings.get('E2E_Enabled_Default_PrivateRooms')) {
```

Comment:
If I understand it correctly, the callback set for `'beforeCreateRoom'` have been ignored in this situation inside the `createPrivateGroup` method. Why?